packages/react-client/src/hooks/useVAD.ts

```diff
@@ -1,9 +1,13 @@
 import { useEffect, useMemo, useState } from "react";
-import type { PeerId, PeerMetadata, TrackId, TrackMetadata } from "../types/internal";
+import type { PeerId, TrackId } from "../types/internal";
 import { useFishjamContext } from "./useFishjamContext";
 import type { TrackContext, VadStatus } from "@fishjam-cloud/ts-client";
 import { useFishjamClientState } from "./useFishjamClientState";
 
+function isMicrophone(metadata: unknown) {
```

Comment:
**note:** this is not needed,  `metadata?.type === "microphone"` is all we need to make sure